test/net.test.js

```diff
@@ -60,7 +60,7 @@ testGetApiVersion = () => {
 testVersions = () => {
     netVersions()
         .then((response) => {
-            assert.deepInclude(response, {'label':'Summer \'19','url':'/services/data/v46.0','version':'46.0'}, 'Wrong version response');
```

Comment:
Should it have '20' and '49.0' for assertion content?

Reply:
Good catch! I didn't run the test :-(